lib/Doctrine/Migrations/Configuration/Configuration.php

```diff
@@ -174,23 +190,45 @@ public function getQuotedMigrationsExecutedAtColumnName() : string
             ->getQuotedName($this->connection->getDatabasePlatform());
     }
 
+    /**
+     * @deprecated Use addMigrationsPath()
+     */
     public function setMigrationsDirectory(string $migrationsDirectory) : void
     {
         $this->migrationsDirectory = $migrationsDirectory;
     }
 
+    /**
+     * @deprecated Use getMigrationsDirectory()
```

Comment:
Looks wrong to me, when this Method is deprecated I cannot use the same method instead.

Reply:
Hi, Thanks for the review, but your comment is based on an outdated file (as highlighted by github).
I suggest to do a review on the latest version of this PR visible from the [full diff](https://github.com/doctrine/migrations/pull/858/files) or to checkout this PR locally by cloning https://github.com/goetas/migrations.